### PR TITLE
Make Promo Banner a Carousel

### LINKED ISSUE #74: Make the promo banner a carousel
We have this really cool promo banner at the top of every page that currently displays the show of the month:

screen shot 2017-03-08 at 3 04 41 pm

Since there's a lot of cool stuff radio does, wouldn't it be cool if we could have the promo banner should multiple images/links?

There are already a lot of carousel plugins out there, so this shouldn't be that hard to do: the key will be integrating it with the rest of the site. Check out [`react/frontpage/Frontpage.jsx`](https://github.com/uclaradio/uclaradio/blob/master/react/frontpage/Frontpage.jsx), the promo banner code is in there.

### PR DESCRIPTION
**Pull request by binerys**
Fixes #74 
* Uses [react-slick](https://github.com/akiran/react-slick) for the carousel 

* Currently the images are hard coded in the new `PromoBanner.jsx` component, I might update this so the data is set in a JSON array so it's cleaner

Update: Commit [91b703a](https://github.com/uclaradio/uclaradio/pull/77/commits/91b703a09b7d1252293b5525874c51dc011790e8) addresses above

screen shot 2017-03-13 at 12 30 25 am
screen shot 2017-03-13 at 12 30 31 am

### CODE REVIEW

**Comment by nathunsmitty** [comment]
I really like this, especially the little arrows on each side! Good job @binerys.

The bounds of hovering over the promo banner were a little off, which I fixed in [7ece85d](https://github.com/uclaradio/uclaradio/pull/77/commits/7ece85d1ab3d789ee1f91ce77d1934927339ae16).

Also, I noticed this on the console:

screen shot 2017-03-13 at 2 53 46 pm

Maybe look into that?

Other than that, it looks good to me. 🙂

**Comment by ChrisLaganiere** [comment]
Very cool :) Note - those warnings in the browser aren't bree's fault, those are errors in `vendors.js` which get resolved when you compile with the correct environment variables:
```
$ export NODE_ENV=production
$ grunt
...
```

File: react/frontpage/components/PromoBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <Link to={banner.link}>
+            <RectImage src={banner.img} aspectRatio={5}>
+              <div className="overlay" />
+            </RectImage>
+          </Link>
+        </div>
+      ); 
+    });
+
+    console.log(banners);
```
Comment (ChrisLaganiere) [inline react/frontpage/components/PromoBanner.jsx]: maybe remove test print statements?

Comment (binerys) [inline react/frontpage/components/PromoBanner.jsx]: Thank's for catching that, will do. And I'll also try out the production env!

**Comment by binerys** [comment]
@nathunsmitty  thanks for adding that fix!

**Pull request merged**